x/nft/blockchain/codec.proto

```diff
@@ -12,14 +13,26 @@ message BlockchainToken {
 }
 
 message TokenDetails {
-    repeated Network networks = 1 [(gogoproto.nullable) = false];
+    Config config = 1 [(gogoproto.nullable) = false];
 }
 
-message Network { // these are all placeholders
-    bytes tickers = 1;
-    bytes bootstrap = 2;
-    bytes codec = 3;
-    bytes codecSettings = 4;
+enum Codec {
```

Comment:
q: by using a enum for this we would need to deploy a new backend for any new type. How about using a string type instead, to be more flexible?

Reply:
From what I understand by the spec, we will curate the list of blockchains. Therefore we are in full control what we let in and what not. IMHO using a enum type will slow us down and put unnecessary release pressure on the backend team. I strongly suggest using a string type instead.

Reply:
I'd like some more input here as this is not what I understood from the discussion about validating payloads. I realise how that would affect release procedure, but if we do indeed need to list/validate each codec config payload - it makes total sense to have this as an enum.

To make this decision we need to know the use-cases of adding new codecs, validating them on the backend (or not) etc.

Reply:
We should opt for a string type. If someone submits a bad codec type thats on them.